indico/modules/events/papers/controllers/management.py

```diff
@@ -71,3 +88,23 @@ def _process_PUT(self):
     def _process_DELETE(self):
         set_reviewing_state(self.event_new, request.view_args['reviewing_type'], False)
         return jsonify_data(flash=False, html=_render_paper_dashboard(self.event_new))
+
+
+class RHManageCompetences(RHManagePapersBase):
+    """Manage the competences of the call for papers ACLs"""
+
+    def _process(self):
+        form_class = make_competences_form(self.event_new)
+        form = form_class(obj=FormDefaults(get_user_competences(self.event_new)))
+        if form.validate_on_submit():
+            user_competences = self.event_new.cfp.user_competences
+            user_competences_data = {int(key[12:]): value for key, value in form.data.iteritems()}
```

Comment:
I'd add `if key.startswith('competences_')` to make it clear what the `[12:]` is about